### ISSUE TITLE
CMA1

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/fmi-standard#429]
**Reported by edrenth on 12 Jan 2018 17:47 UTC**
Energy preserving iterative co-simulation can be achieved by inserting a CMA1 anti-aliasing filter. Please read further on FCP_013_CMA1 for the rationale.

----

Migrated-From: https://trac.fmi-standard.org/ticket/429

**Comment by modelica-trac-importer** [comment]
**Comment by cbertsch on 1 Jun 2018 15:26 UTC**
To be discussed if this should be part of the FMI standard or if it is only a modeling or master tool issue.

**Comment by chrbertsch** [comment]
As the content of this proposal is not _necessaryly_ needed to implement a future FMI  standard that would contain this  proposal, to my understanding of the CCLA there is no automatic granting of a patent license for a required patent  of Volvo in this case (as the CCLA does not apply for optional features) without an additional legally binding statement explicitly granting a patent license.

citing form the CCLA
> where such license applies only to those patent claims licensable by You that are _necessarily_ infringed by Your Contribution(s) alone or by combination of Your Contribution(s) with the Work to which such Contribution(s) were submitted.

**Comment by chrbertsch** [comment]
Closing for the following reasons: 
- CMA1 can be realized with Intermediate Variable Access that is already part of the master branch.
- It specifies a certain master algorithm (partially realized at FMU export by including a certain filter into the exported FMU. However, this issue tracker is about the FMI Standard -  describing only the _Interface_ So, @edrenth: If you have further proposals going beyond the Intermediate Variable Access on the interface: please open a separate ticket.
- as stated in the previous comment: there is a patent application pending on this topic, and submitting it from Volvo under the  CCLA is not sufficient to get a license granted (as this features is not necessary to implement the FMI standard, but and optional feature)

If someone disagrees, please reopen
